models/migrations/v1_24/v314.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_24 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+func AddEphemeralToActionRunner(x *xorm.Engine) error {
+	type ActionRunner struct {
+		Ephemeral bool `xorm:"ephemeral"`
+	}
+
+	return x.Sync(new(ActionRunner))
```

Comment:
It's better to update the legacy data as `false` rather than `null`.

Reply:
I wonder if you mean something like that
```suggestion
	if err := x.Sync(new(ActionRunner)); err != nil {
		return err
	}

	// update all records to set ephemeral to false
	_, err := x.Exec("UPDATE `action_runner` SET `ephemeral` = false WHERE `ephemeral` IS NULL")
	return err
```

Or adding a xorm tag like this for both migration and model (I'm not 100% sure, no examples in Gitea has both name and the not null sequence)
```
Ephemeral bool `xorm:"ephemeral NOT NULL DEFAULT false"`
```

**EDIT** using second variant, but needs to be tested if it works or not.
First variant is broken for mssql: ``SET `ephemeral` = false``